lib/container_orchestrator/object_definition.rb

```diff
@@ -2,18 +2,34 @@ class ContainerOrchestrator
   module ObjectDefinition
     private
 
+    def common_labels
+      {:app => app_name}.merge(orchestrated_by_label)
```

Comment:
Might want to extract `{:app => app_name}` into app_name_label for consistency...then you can also use it elsewhere in the definitions.